src/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj

```diff
@@ -30,7 +30,6 @@
     <ProjectSystemLayer>VisualStudio</ProjectSystemLayer>
     <ProducingSignedVsix>true</ProducingSignedVsix>
     <IsProductComponent>true</IsProductComponent>
-    <InstallRoot>Default</InstallRoot>
```

Comment:
Ah, my commit was a little misleading, by "default setting" I mean we setting InstallRoot to it's default value, which happens to be "Default". According to Alex, InstallRoot set to Default *is the* default.